aws/src/test/java/org/apache/iceberg/aws/glue/TestIcebergToGlueConverter.java

```diff
@@ -216,6 +229,10 @@ public void testSetTableInputInformationWithRemovedColumns() {
             .build())
         .build();
 
+    Assert.assertEquals(
+        "additionalLocations do not match",
```

Comment:
the message here should describe the expected behavior, instead of the wrong behavior, like `"additionalLocations should match"`. It seems like we are also doing this wrong at L237 and L241, could you update those messages as well?